ci: add packaging scripts for npm (#2047)
googleapis/genai-toolbox: npm/server/README.md

> # Toolbox

Reviewer: ```suggestion
# MCP Toolbox for Databases
```

A CLI tool for running a toolbox server.

## Installation

You can install the toolbox globally:

> ```bash
> npm install -g @toolbox-sdk/server
> ```
> 
> Or run it directly using npx:
> 
> ```bash
> npx @toolbox-sdk/server
> ```
> 
> ## Configuration
> 
> The toolbox requires a `tools.yaml` file in the current working directory to define sources, tools, and prompts.

Reviewer: ```suggestion
## Configuration

Toolbox supports two ways to define tools: **Prebuilt Sources** for instant database access and **Custom YAML** for tailored logic.

### 1. Prebuilt Tools
Use the `--prebuilt` flag to skip configuration files and instantly expose standard database operations.

*   **Usage:** `npx @toolbox-sdk/server --prebuilt <source> --stdio`
*   **Supported:** [See all supported datasource](https://mcp-toolbox.dev/documentation/configuration/prebuilt-configs/).
*   **Setup:** Requires relevant environment variables (e.g., `BIGQUERY_PROJECT` or `POSTGRES_HOST`).

### 2. Custom Tools (`tools.yaml`)
The toolbox requires a `tools.yaml` file in the current working directory to define sources, tools, and prompts.

*   **Usage:** The server loads `tools.yaml` automatically, or use `--config <path>`.
*   **Function:** Defines `sources` (connection details) and `tools` (specific SQL/logic with descriptions for the LLM).
```

> ### Example `tools.yaml`

Reviewer: ```suggestion
#### Example `tools.yaml`
```
